openapi-bal-task-plugin/src/main/java/io/ballerina/openapi/bal/tool/OpenAPICodeGeneratorTool.java

```diff
@@ -190,10 +213,16 @@ private OpenAPI getOpenAPIContract(Path ballerinaFilePath, Path openAPIPath, Loc
                 File openapiContract = new File(file, openAPIPath.toString());
                 relativePath = Paths.get(openapiContract.getCanonicalPath());
             }
-            return normalizeOpenAPI(Path.of(relativePath.toString()), true);
+            if (Files.exists(relativePath)) {
+                return normalizeOpenAPI(Path.of(relativePath.toString()), true);
+            } else {
+                Constants.DiagnosticMessages error = Constants.DiagnosticMessages.INVALID_CONTRACT_PATH;
+                toolContext.reportDiagnostic(createDiagnostics(error, error.getDescription(), location));
+            }
```

Comment:
We are already doing an early return if `Files.exists(relateivePath)` evaluates to true. So, I think we do not need the `else` block here as it would add unnecessary indentation.

Reply:
Yeah we can, will do with early return.